src/sagas/collection.ts

```diff
@@ -53,7 +53,7 @@ export function* listRecords(
       limit: MAX_PER_PAGE,
     });
     // Show list of records.
-    yield put(actions.listRecordsSuccess(data, hasNextPage, next));
+    yield put(actions.listRecordsSuccess(data, hasNextPage, next, false));
```

Comment:
nitpick: can we add a little annotation for this `false`? like:
```suggestion
    yield put(actions.listRecordsSuccess(data, hasNextPage, next, /* isNextPage: */ false));
```